### PR TITLE
Release preparations

### PR DESCRIPTION
**Pull request by cnavacch**
Includes: 

- first sketch of a metadata class
- some tests
- metadata config files for SIG0 and PLIA
- complete docstrings

### CODE REVIEW

**Review by AlenaDostalova — COMMENTED** [review]
Hi, the geometry tag should not be contained in the metadata tags in geotiff. It will be in the metadatabase, but it is basically description of the file position done by the database software (polygon written as wkt string). The geotiff has its own information about geolocation. Also, I think that it does not make sense to store filename and path of the file as the hardcoded tags in the geotiff itself (although they are of course stored in the metadata). I also left out tile_id as I couldn't make it work to write it to the geotiff metadata (as I failed to write some additional metadata in the resampling step where this needs to be done), so if you want to include it, check first, how to implement this information into the data. Otherwise, changes are fine with me.

File: src/medali/core.py [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+            elif self._ref_meta['Metadata'][attribute] == 'datetime':
+                value = datetime.datetime.strptime(self._meta[attribute], "%Y-%m-%d %H:%M:%S")
+            else:
+                value = self._meta[attribute]
+        else:
+            value = self._meta[attribute]
+
+        return value
+
+    def check_metadata(self, metadata):
```
Comment (fl0roth) [inline src/medali/core.py]: While initializing the MetaData class there is already a metadata dictionary read in which contains decoded values, while in this function a metadata dictionary with encoded values is needed. Could we use the already existing dictionary here instead?

Comment (cnavacch) [reply src/medali/core.py]: Yes, but then we need more effort on decoding and encoding the attributes and would need to change the class interaction with metadata coming from a file, in string format. I will try to change this, that the class becomes more compact.

Comment (fl0roth) [reply src/medali/core.py]: In my opinion it is not a big deal to leave it like this. I just wanted to know if it is maybe only a mistake which could be fixed easily.

**Pull request merged**